openapi/components/schemas/TagUntag/TagUntagRule.yaml

```
@@ -0,0 +1,94 @@
+type: object
+description: Tag / Untag rule.
+required:
+  - name
+  - eventType
+  - addingTags
+  - removingTags
+properties:
+  id:
+    description: Unique rule identifier string.
```

nitpick: it's better to use 'ID' (based on our writing style)
```suggestion
    description: ID of the rule.
```